wrappers/python/pyrs_frame.cpp

```diff
@@ -4,6 +4,8 @@ Copyright(c) 2017 Intel Corporation. All Rights Reserved. */
 #include "pyrealsense2.h"
 #include <librealsense2/rs.hpp>
 
+#include <src/image.cpp>  // bad idea? for get_image_bpp
```

Comment:
Including image.h won't work?

Reply:
No, the function is not exposed thru rs2. I needed the actual implementation as part of the pyd.

Reply:
OK. I don't like including cpp files but it is in the python wrapper so lesser evil.
Exposing it in the API is currently not necessary.